TLM/TLM/TLM.csproj

```diff
@@ -570,10 +567,15 @@ xcopy /y "$(TargetDir)CSUtil.CameraControl.dll" "%25DEPLOYDIR%25"
 xcopy /y "$(TargetDir)TMPE.RedirectionFramework.dll" "%25DEPLOYDIR%25"
 xcopy /y "$(TargetDir)CSUtil.Commons.dll" "%25DEPLOYDIR%25"
 xcopy /y "$(TargetDir)0TMPE.Harmony.dll" "%25DEPLOYDIR%25"
+
+REM to avoid double hot reload, TrafficManager.dll must be replaced last and fast. 
```

Comment:
`REM to avoid...` reads like an instruction, uppercase `T` would help disambiguate:

`REM To avoid...`